src/main/java/org/jabref/gui/preferences/PreferencesDialogView.java

```diff
@@ -59,6 +59,10 @@ public PreferencesDialogViewModel getViewModel() {
         return viewModel;
     }
 
+    public ListView<PreferencesTab> getPreferenceTabList() {
+        return preferenceTabList;
+    }
```

Comment:
Do not touch the PreferencesDialog. Changes must happen in the Preferences, not in the Dialog

Reply:
Or perhaps make the method more generic, so a new method could open any specific preferences tab.